client/src/app/site/pages/meetings/view-models/view-meeting.ts

```diff
@@ -133,6 +133,10 @@ export class ViewMeeting extends BaseHasMeetingUsersViewModel<Meeting> {
     public default_projectors(place: ProjectiondefaultValue): ViewProjector[] {
         return this[`default_projectors_${place}`];
     }
+
+    public canBeEnteredBy(userId: number): boolean {
+        return !this.locked_from_inside || this.user_ids?.includes(userId);
```

Comment:
The only reference to this seems to be in `committee-meeting-preview.component.ts` which already does a check if the user is with a meeting. Why do you need to check this here again?

Reply:
Use another way to get the meeting membership. S.o.